keyboards/panc60/readme.md

```diff
@@ -14,7 +14,11 @@ Make example for this keyboard (after setting up your build environment):
 
 Flashing
 
+<<<<<<< HEAD
```

Comment:
Looks like some merge artefacts have managed to sneak themselves in.

Reply:
@zvecr if you see any small issues like this, you can actually use the suggestion tool to suggest your fix, that way I can fix it while I'm on my phone lol.

Reply:
As it spans over 2 blocks, i figured you would want to tackle it outside of the suggestion tool to avoid the multiple single line commits. If only it supported a patch suggestion.

Reply:
Lack of patch suggestion still irritates me.